lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -42,6 +42,15 @@ class KubernetesMetadataFilter < Fluent::Filter
     config_param :secret_dir, :string, default: '/var/run/secrets/kubernetes.io/serviceaccount'
     config_param :de_dot, :bool, default: true
     config_param :de_dot_separator, :string, default: '_'
+    # if reading from the journal, the record will contain the following fields in the following
+    # format:
+    # CONTAINER_NAME=k8s_$containername.$containerhash_$podname_$namespacename_$poduuid_$rand32bitashex
+    # CONTAINER_FULL_ID=dockeridassha256hexvalue
+    config_param :journal_container_name, :string, default: 'CONTAINER_NAME'
```

Comment:
Why do you need this? It is supposed to be 'CONTAINER_NAME' always, right?
on line 237 you use 'CONTAINER_NAME', not journal_container_name